web/meetup-facilitator/.gitignore

```diff
@@ -22,3 +22,5 @@ dist-ssr
 *.njsproj
 *.sln
 *.sw?
+
+.env
```

Comment:
Could you add the contents of the `.env` to our secrets doc?

Reply:
I have added the contents of the `.env` to the secrets doc. Regarding the above comment, does that mean I just have to get the approval and merge the other PR first?